ydb/library/yql/parser/pg_wrapper/ut/arrow_ut.cpp

```diff
@@ -0,0 +1,116 @@
+#include <arrow/api.h>
+#include <arrow/array.h>
+
+#include <library/cpp/testing/unittest/registar.h>
+
+#include "arrow.h"
+
+extern "C" {
+#include "utils/numeric.h"
+
+Datum numeric_out(PG_FUNCTION_ARGS);
+}
+
+namespace NYql {
+
+Numeric PgFloatToNumeric(double item, ui64 scale, int digits);
```

Comment:
please extract this to .h instead of redeclaration

Reply:
Fixed